app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/ContributorsFragment.java

```diff
@@ -41,7 +63,17 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         return binding.getRoot();
     }
 
+    @Override
+    public void onStop() {
+        super.onStop();
+        compositeDisposable.dispose();
+    }
+
     private void initView() {
         //
     }
+
+    private void renderContributors(List<Contributor> contributors) {
+        // TODO render contributors
```

Comment:
On next PR, can you bind `LIst<ContributorViewModel>` (not `List<Contributor>`) to RecyclerView?